daemon/src/maker_cfd_actor.rs

```diff
@@ -285,13 +285,12 @@ impl MakerCfdActor {
         self.cfd_feed_actor_inbox
             .send(load_all_cfds(&mut conn).await?)?;
 
-        // 4. Start contract setup
+        // Start contract setup
         tracing::info!("Starting contract setup");
 
         // Kick-off the CFD protocol
         let (sk, pk) = crate::keypair::new(&mut rand::thread_rng());
 
-        let cfd = load_cfd_by_order_id(order_id, &mut conn).await?;
```

Comment:
So actually, we should do `cfd.state = CfdState::Accepted` here.

Reply:
I guess that would be cleaner...
 
It's not crucial though because the state is not used and a few lines below we update the db with the new state `CfdState::ContractSetup`. 

Shall I add it nevertheless?

Reply:
Not necessary then :)